static/templates/settings/organization-permissions-admin.handlebars

```diff
@@ -95,11 +95,14 @@
 
 
             <div class="input-group disableable">
-                <label for="aitin">{{t "Waiting period for stream creation (in days)" }}</label>
-                <input type="text" id="id_realm_waiting_period_threshold"
-                     name="realm_waiting_period_threshold"
-                     class="admin-realm-message-content-edit-limit-minutes"
-                     value="{{ realm_waiting_period_threshold }}"/>
+                <label for="aitin">
+                    {{t "Prevent new users from creating stream for" }}
+                    <input type="text" id="id_realm_waiting_period_threshold"
+                         name="realm_waiting_period_threshold"
+                         class="admin-realm-message-content-edit-limit-minutes"
+                         value="{{ realm_waiting_period_threshold }}"/>
+                    {{t "days" }}
```

Comment:
`./tools/check-capitalization` tests are failing, cause it's not capitalized. But it is not supposed to capitalized, as it is not new statement but last of part of statement. I have checked [here](https://zulip.readthedocs.io/en/latest/translating/translating.html#capitalization), to ignore string for this test. But that solution doesn't sound good to me.
Any other ideas about how I can change it in a way it pass the test?

Reply:
Should there be a space before "days" (" days")?  That might bypass the linter and communicate to people reading the code that you're finishing a sentence, not starting one.

Reply:
Space before "days" doesn't pass linter tests, but I change it to "(days)", in parenthesis. Is it ok?